distribution/std-lib/Standard/src/Visualization/Helpers.enso

```diff
@@ -1,8 +1,37 @@
 from Standard.Base import all
 
+import Standard.Table.Data.Column
+import Standard.Table.Data.Storage
+import Standard.Table.Data.Table
+
+## PRIVATE
+Any.catch_ ~val = this.catch (_-> val)
+
+## PRIVATE
+Error.catch_ ~val = this.catch (_-> val)
+
 ## PRIVATE
 recover_errors ~body =
     result = Panic.recover body
     result.catch err->
         Json.from_pairs [["error", err.to_display_text]] . to_text
 
+## PRIVATE
+
+   Returns all the columns in the table, including indices.
+   Index columns are placed after other columns.
```

Comment:
Please wrap this at 80 rather than on sentences.

Why the decision to place index columns after the other columns? My instinct would've been to put them before.

Reply:
Incidentally this is the order that scatter plot logic wants them in such order. Then histogram needs all columns in any order —so I've put method with this order to helpers, rather then implement both variants.

Reply:
But I will split them.